viadot/tasks/tm1.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+
+from prefect import Task
+from typing import Any, Dict
+from prefect.utilities.tasks import defaults_from_attrs
+
+from ..sources import TM1
+
+
+class TM1ToDF(Task):
+    def __init__(
+        self,
+        credentials: Dict[str, Any] = None,
+        config_key: str = "TM1",
+        mdx_query: str = None,
+        cube: str = None,
+        view: str = None,
+        limit: int = None,
+        private: bool = False,
+        verify: bool = False,
+        if_empty: str = "skip",
+        timeout=3600,
+        *args,
+        **kwargs,
+    ):
+        """
+        Task for downloading data from TM1 view to pandas DataFrame.
```

Comment:
Please add here information that you need either `query` or `view + cube` and not necessarily both to be able to use this connector.